### DISCUSSION TITLE
Commits Number not correct after switching branches

### DISCUSSION THREAD

**Discussion by JeremyGrieshop** [community/community#3454] [Mobile] [answered]
I noticed this in the most recent iOS GitHub app update.  After switching branches, the total commit number didn't change from the previous branch.

**Comment by brianlovin** [comment, marked as answer]
Hey @JeremyGrieshop, apologies for the slow reply here. Is this still a bug for you? I can't repro on my end, when I switch branches I'm seeing the commit counts updating to reflect the same values that we have on the web interface.

**Reply by JeremyGrieshop** [reply]
I believe it was fixed in the most recent update.  The change log mentioned it and I confirmed it was fixed.

**Reply by brianlovin** [reply]
Great! Cool to mark this thread as answered so we can close it out?

**Reply by JeremyGrieshop** [reply]
Yes, and thank you!
